client/src/main/java/org/evosuite/coverage/mutation/MutationTestFitness.java

```diff
@@ -48,7 +48,7 @@ public abstract class MutationTestFitness extends TestFitnessFunction {
 
     private static final long serialVersionUID = 596930765039928708L;
 
-    protected transient Mutation mutation;
+    protected Mutation mutation;
```

Comment:
Why does not the `mutation` field need to be `transient`?

Reply:
We had to make the "Mutation" class Serializable, so the "mutation" field does not need to remain transient.